src/Calculator.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import NumberInput from './NumberInput'
+import OperatorSelect from './OperatorSelect'
+import Equal from './Equal.js'
```

Comment:
This component isn't being used in this file so it isn't necessary to import it.